docs/Gemfile

```diff
@@ -15,3 +15,5 @@ group :jekyll_plugins do
 # Delete the following lines if not on Windows:
 # Performance-booster for watching directories on Windows
 gem "wdm", ">= 0.1.0" if Gem.win_platform?
+
+gem "webrick", "~> 1.7"
```

Comment:
@AnshumanDhiman Could you please revert this change? Thanks for your contribution 🎉